doc/en/user/source/styling/sld/working.rst

```diff
@@ -35,7 +40,22 @@ using the :ref:`data_webadmin_layers` page of the :ref:`web_admin`.
 .. note:: When adding a layer and a style for it to GeoServer at the same time, the style should be added first, 
           so that the new layer can be associated with the style immediately. 
 
-          
+Style Groups
```

Comment:
Same comment as above; not sure if this is clearer from a user perspective, especially since we refer to the styling language as Style Layer Descriptor.